addons/stock/models/stock_move_line.py

```diff
@@ -64,9 +64,9 @@ class StockMoveLine(models.Model):
         help="When validating the transfer, the products will be taken from this owner.")
     location_id = fields.Many2one(
         'stock.location', 'From', domain="[('usage', '!=', 'view')]", check_company=True, required=True,
-        compute="_compute_location_id", store=True, readonly=False, precompute=True,
+        compute="_compute_location_id", store=True, readonly=False, precompute=True, index='btree_not_null',
```

Comment:
This one is required, then it should be btree

Reply:
Good catch, I've missed it, will update.